### PR TITLE
Navbar to collapse when clicked on

### PR DESCRIPTION
**Pull request by julietrinh1**
- Added an onClick and a constant to keep track of when to expand or not the navbar.

### CODE REVIEW

**Review by kalsij — APPROVED** [review]
Ok looks good

File: frontend/src/components/Navbar.js [hunk 1]
```diff
@@ -12,13 +12,19 @@ import bell from "./../images/icon_bell.png";
...
 
 import Search from "./connection/Search";
 
 function NavbarFun() {
   const userr = auth.currentUser;
   const { logOut } = useUserAuth();
   const navigate = useNavigate();
+  const [expanded, setExpanded] = useState(false);
+
+  const handleLinkClick = () => {
```
Comment (Chovv) [inline frontend/src/components/Navbar.js:25]: I'm assuming this is what closes the nav expansion. Looks good to me.

File: frontend/src/components/Navbar.js [hunk 2]
```diff
@@ -44,37 +50,37 @@ function NavbarFun() {
               </Nav>
             </>
           )}
-          <Navbar.Toggle aria-controls="navbarScroll" />
+          <Navbar.Toggle aria-controls="navbarScroll" onClick={() => setExpanded(!expanded)}/>
```
Comment (Chovv) [inline frontend/src/components/Navbar.js:53]: Use of the previously defined handleLinkClink on the navbar on the navbar components. Looking good.

**Review by Chovv — COMMENTED** [review]
Nice, simple and works as intended, well done.

**Pull request merged**